docs/setup_details.md

```diff
@@ -3,65 +3,147 @@ id: setup_details
+## Prerequisites
 
+It is important that your computer is set up with the necessary packages before you begin development. You will come across a variety of errors if these prerequisites are not installed. First we need the following programs
 
+- git (version control and GitHub integration)
+- nodejs (web development - we recommend the latest long term support version)
+- python (3.7 or later)
+- An Integrated Development Environment (we recommend VS Code)
 
-### Help With Installing [Electron Prerequisites](https://brown-ccv.github.io/honeycomb-docs/docs/quick_start#0-installing-prerequisites) 
+In addition, Honeycomb relies on Electron to package the cross-platform desktop applications. Some of electron's prerequisites are platform dependant - listed below are the prerequisites for each operating system along with resources for manually installing them.
 
-Honeycomb relies on Electron to package the cross-platform desktop applications.
-You will need to install Electron's pre-requisites below. You can find the full instructions on the [electron documentation](https://www.electronjs.org/docs/development/build-instructions-gn) for your specific OS.
+## MacOS
 
-Listed below are the prerequisites for each operating system along with resources for manually installing them.
+### Brew
 
-#### Windows
+All of the required dependencies for honeycomb on MacOS can be installed via the brewfile. See the [quick start](quick_start.md/#installing-prerequisites-with-homebrew-for-macos) for more information.
 
-- Visual Studio:
-  Install the latest version of [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the Desktop Development for C++ Workflow.
-  To add the workflow, follow [these instructions](https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-160#:~:text=If%20you%20have%20Visual%20Studio,Then%2C%20choose%20Modify).
+### Manual Installation (MacOS)
+
+- [XCode](https://developer.apple.com/xcode/) must be installed from the terminal:
 
-- Node.js:
-  Install [Node.js](https://nodejs.org/en/download/)
+  ```terminal
+    xcode-select --install
+  ```
 
-- Git bash (Git 2.20.0 or later with support for "--show-current"):
-  Install [git bash](https://git-scm.com/downloads)
+- [Git 2.20.0 or later](https://git-scm.com/downloads/) with support for "--show-current"
+- [Node.js](https://nodejs.org/en/download/)
+- To install Python (3.7 or later with support for TLS 1.2) follow the guide on electron docs to [install and configure python and its modules](https://www.electronjs.org/docs/development/build-instructions-macos#python)
 
-**Note: restart computer after all installs are complete**
+## Windows
 
-#### MacOS
+Note that Windows occasionally requires a restart after package installations complete. If you are running into issues after installing the packages please restart your terminal and/or your entire computer. This should resolve most issues.
 
-- Command Line Tools: Type in the terminal
+### chocolatey
```

Comment:
Maybe a sentence on what chocolatey is?

Reply:
I have a sentence in the quick start, should I just copy it here as well?

Reply:
Oh is it already in the quick start? Ehh should be fine then !